app/Http/Controllers/Api/AlbumController.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Http\Resources\Album as AlbumResource;
+use App\Http\Resources\AlbumCollection;
+use App\Models\Album;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Symfony\Component\HttpFoundation\Response;
+
+class AlbumController extends Controller
+{
+    public function get(Album $album)
+    {
+        return new AlbumResource($album);
+    }
+
+    public function all()
+    {
+        return new AlbumCollection(Auth::user()->albums()->paginate());
+    }
+
+    public function update(Request $request, Album $album)
+    {
+        $validator = validator($request->only('name', 'description', 'isDownload', 'visibility', 'sort'), [
+            'name' => 'required|string|max:255',
+            'description' => 'required|string|nullable|max:1000',
+            'isDownload' => 'required|boolean',
```

Comment:
`downloadable` ?

Reply:
Yeah it's on my todo to change that attribute name.